stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/model/accessor/MethodFieldAccessor.java

```diff
@@ -104,7 +104,7 @@ private static String findSetterMethodName(@NotNull VariableElement variableElem
 
         }
 
-        throw new UnsupportedOperationException("Unable to find setter for variable");
+        throw new UnsupportedOperationException("Unable to find setter for variable: " + variableElement.getSimpleName());
```

Comment:
There are various lint warnings in this class related to final that can be cleaned up.